src/core/lib/iomgr/executor.cc

```diff
@@ -409,6 +412,10 @@ void Executor::Run(grpc_closure* closure, grpc_error* error,
                        [static_cast<size_t>(job_type)](closure, error);
 }
 
+bool Executor::CurrentThreadCanShutdownAll() {
+  return 0xdeadbeef != gpr_tls_get(&g_this_thread_is_executor_thread);
```

Comment:
We generally don't use yoda comparisons. Also, why not just 0 or 1?

Reply:
Fixed.